Documentation/Functions/Typolink.rst

```diff
@@ -428,7 +428,18 @@ returnLast
     So, in these two cases you will not get the value wrapped but the URL
     or target value returned!
     
-    If set to ``result``, it will return the ``LinkResult`` object (only since TYPO3 11.4)
+    If set to ``result``, it will return the json_encoded output of the ``LinkResult`` object (only since TYPO3 11.4)
```

Comment:
for me is the info in the round brackets not clear.
Does the parameter value `result` exist since TYPO3 v11.4 or was the output just differently before?

For the first case I propose this:
```
If set to ``result`` (available since TYPO3 11.4), it will return the json_encoded output of the ``LinkResult`` object.
```
Not related to this issue in particular: @linawolf is it possible that with changed rights I could propose with the code icon so that my propositions could be confirmed and applied with a single click?

Reply:
We should use the directive `.. versionadded:: 11.4`.

Reply:
@brotkrueml, alright, think I got it now :muscle: Please squash it when merging ;)